packages/ui/src/styles/global/_typography.scss

```diff
@@ -192,6 +192,10 @@ ol.va-ordered {
   @include va-table();
 }
 
+.ag-theme-vuestic {
```

Comment:
Check the class name in the docs. It's `ag-grid-vuestic` there instead of `ag-theme-vuestic`

Reply:
fixed